src/js/components/MosaicView.js

```diff
@@ -6,24 +6,55 @@ import {
   metadataType,
   mosaicSingleView,
 } from "./componentPropTypes"
+import { _getById, addStrHashCode } from "../modules/utils"
+
+const FRAMELENGTH = 0.1
+const BACK = -FRAMELENGTH
+const FORWARD = FRAMELENGTH
+
+// add hashing method to String prototype
+addStrHashCode()
 
 const commonColumns = ["seqNum"]
 
 export default function MosaicView({ locationName, camera }) {
   const [historicalBusy, setHistoricalBusy] = useState(null)
   const [currentMeta, setCurrentMeta] = useState({})
-  const initialViews = camera.mosaic_view_meta.map((view) => ({
-    ...view,
-    latestEvent: {},
-  }))
+  const isFocusable = checkNeedsFocusability()
   const [views, setViews] = useState(initialViews)
 
+  function initialViews() {
+    const views = camera.mosaic_view_meta.map((view, index) => ({
+      ...view,
+      latestEvent: {},
+      hasFocus: index == 0 ? true : false,
+    }))
+    return views
+  }
+
+  function checkNeedsFocusability() {
```

Comment:
I recommend to change the name of this function per something more related to what the function is doing, like:
```suggestion
  function checkMoreThanOneImage() {
```
Then if there is a second condition to set the `isFocusable`, you can arrange it like:
```
const isFocusable = checkMoreThanOneImage() && checkSecondCondition()
```